packages/next/pages/_app.tsx

```diff
@@ -63,6 +68,16 @@ export default class App<P = {}, CP = P> extends React.Component<
 export class Container extends React.Component {
```

Comment:
I think we should make container a noop and move the code into `client/index.ts`